### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlfredHellstern**
update to account for package version on nuget

### CODE REVIEW

**Review by tomjebo — DISMISSED** [review]

**Review by mikeebowen — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Open XML SDK
...
 > 
-> v3.0.0-Beta1 will refactor a number of things to address some technical debt, with the goal being to retain (as much as possible) source compatibility. This means, we hope to maintain things such that you can update your package and recompile with limited changes. However, binary compatibility will not be a goal and will break that for some changes. However, we plan on keeping these documented. PRs that introduce such changes will be marked with a `breaking-change` label and will be added to a list to help migrating to v3.0.0. These changes will occur on the `main` branch.
+> Version 3.0 Beta 1 (the package version is `v3.0.0-ci0083` on nuget) refactors and addresses some technical debt, with the goal to retain (as much as possible) source compatibility. We hope to maintain things such that you can update your package and recompile with limited changes. However, binary compatibility is not a goal and will break that for some changes which we plan on keeping documented. PRs that introduce such changes are marked with a `breaking-change` label and are added to a list to help migrating to v3.0.0. These changes will occur on the `main` branch.
```
Comment (twsouthwick) [inline README.md:9]:
```suggestion
> Version 3.0 Beta 1 (the package version on NuGet is `v3.0.0-ci0083`) refactors and addresses some technical debt, with the goal to retain (as much as possible) source compatibility. We hope to maintain things such that you can update your package and recompile with limited changes. However, binary compatibility is not a goal and will break that for some changes which we plan on keeping documented. PRs that introduce such changes are marked with a `breaking-change` label and are added to a list to help migrating to v3.0.0. These changes will occur on the `main` branch.
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Open XML SDK
 > 
 > The v2.20.0 release (`release/v2.20.0`) is limited in scope and focuses on obsoleting types that will be removed in v3.0.0. This provides a stepping stone for migrating to v3.0.0.
 >
```
Comment (twsouthwick) [inline README.md:8]:
```suggestion
```

Comment (twsouthwick) [reply README.md:8]: no longer needed

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**